### ISSUE TITLE
Create camel-sensor-dataformat module

### ISSUE THREAD

**Issue by hekonsek** [camel-labs/camel-labs#60]
We should come up with some opinionated data formats that can be used to send metrics from the sensors to data center. The data formats should include information like message id, client id, timestamp, metrics name, metrics value, etc.

**Comment by gautric** [comment]
@hekonsek any evolution ? I need it for i2c  data struct for transportation

**Comment by hekonsek** [comment]
Yes, I have some stuff ready on my local branch. I should push it in coming days. Can you wait so long?

**Comment by gautric** [comment]
we need agregator data format (gpsvalue, tempvalue, accelvalue)
and data format for all kind of metric (component independant)
GPSData (X,Y,Z,T,Unit {W84})
TEMPData (Value,  T,Unit  {C,K,F})
ACCELData(X,Y,Z,T,Unit  {mG})
Humidity(Value,Unit {%})

**Comment by hekonsek** [comment]
Actually we might think about some generic key/value pairs data format.
What do you think?

**Comment by gautric** [comment]
any idea of cool impl ?

**Comment by davsclaus** [comment]
there is the "cool" java.util.Properties ;)

**Comment by hekonsek** [comment]
Kinda ;) i was thinking about properties/map to csv (de)serializer. We have
to serialize data structures to some non-chatty payload before sending it
to the wire like MQTT. Csv could be neat.

What do you think?
